app/javascript/stylesheets/lesson-content.css

```diff
@@ -24,33 +24,83 @@
   }
 
   .lesson-note {
-    @apply bg-gray-50 p-4 border-l-4 border-gold dark:bg-gray-700/40 dark:border-gold-700 rounded;
+    @apply bg-gray-50 p-4 my-4 border-l-4 border-gold dark:bg-gray-700/40 dark:border-gold-600 rounded;
   }
 
-  .lesson-note > h4::before {
+  .lesson-note > *:first-child::before {
     font-family: "Font Awesome 6 Free";
+    font-weight: bold;
     content: "\f249";
-    @apply mr-2 text-gold dark:text-gold-700;
+    @apply mr-2 text-gold dark:text-gold-600;
+  }
+
+  .lesson-note > *:only-child::before {
+    display: block;
+    font-family: "Font Awesome 6 Free";
+    font-size: 1.8rem;
```

Comment:
@luuu-xu @thatblindgeye this looks little too big to me in relation to content around it. What do you both think about using something like 1.5? (text-2xl)

1.8
<img width="715" alt="Screenshot 2023-08-09 at 15 06 04" src="https://github.com/TheOdinProject/theodinproject/assets/7963776/d393f7e7-5009-426f-a53f-baa5568edce9">

vs

1.5
<img width="773" alt="Screenshot 2023-08-09 at 15 05 46" src="https://github.com/TheOdinProject/theodinproject/assets/7963776/e2fdd8ac-14b6-4f48-96b3-d200c9d64172">

Reply:
I'm fine with either, as long as the icons are easily recognizable enough. We can always increase/decrease it further if users provide feedback on it

Reply:
1.5 and 1.8 both looks fine to me while 1.5 seems more aesthetically pleasant. Let's do 1.5 first.